lib/omnibus/config.rb

```diff
@@ -400,14 +413,22 @@ def reset!
     #
     # @return [String]
     default(:publish_s3_access_key) do
```

Comment:
@rhass Yeah `Omnibus:: S3Publisher` is for publishing artifacts to S3 after a build completes (like we do with the `Omnibus::ArtifactoryPublisher` but for S3 instead of Artifactory). `Omnibus::S3Cache. populate` is for pushing source archives to the `opscode-omnibus-cache` S3 bucket to make builds more reliable as many of the sites hosting source archives are flakey.